aexpect/ops.py

```diff
@@ -473,3 +477,94 @@ def extract_tarball(session, tarball, target_dir, flags="-ap"):
         output = output.strip()
     if status != 0:
         raise RuntimeError(f'Failed to extract {tarball} to {target_dir}: {output}')
+
+
+###############################################################################
+# system
+###############################################################################
+
+
+def start_services(session, services, timeout=60):
+    """
+    Try starting all services in given list.
+
+    :param session: virtual machine session
+    :type session: :py:class:`aexpect.ShellSession`
+    :param services: list of services to modify
+    :type services: [str]
+    :param int timeout: timeout for each service to start
+    :returns: statuses for each modified service
+    :rtype: [int]
+    """
+    statuses = []
+    for service in services:
+        status = session.cmd_status(f'service {service} status')
+        if status == 0:      # is running --> skip
+            LOG.debug(f'Service {service} already started - skipping')
+        elif status == 3:    # is stopped --> ok
+            LOG.debug(f'Service {service} stopped - starting')
+            status = session.cmd_status(f'service {service} start', timeout)
+        else:                # not sure, try restarting
+            LOG.debug(f'Querying status for service {service} resulted in '
+                      f'unexpected return code {status}. Try starting it')
+            status = session.cmd_status(f'service {service} start', timeout)
+        LOG.debug(f'Starting service {service} resulted in status {status}')
```

Comment:
This is quite chatty, how about combining those messages into one?

    Service {service} was running
    Service {service} was stopped, start returned {status}
    Service {service} status command returned {stat_status}, start returned {status}

And similarly elsewhere?